Chatkit/Persistence/Model/Entities/InstanceEntity+CoreDataProperties.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+import CoreData
+
+extension InstanceEntity {
+    
+    @nonobjc class func fetchRequest() -> NSFetchRequest<InstanceEntity> {
```

Comment:
Out of curiosity - what is the setting for `Codegen`? Are we manually writing NSManagedObjects or is it done automatically by using `Class Definition` setting?

Reply:
It is set to `Manual/None`. Both classes and extensions are generated by Xcode and amended manually for nullability (which is unfortunate, because Xcode should be able to figure this out automatically from the model file).

We can `mogenerator` for that if that is a problem.